spec/requests/case_log_controller_spec.rb

```diff
@@ -180,84 +180,63 @@
           allow(FormHandler.instance).to receive(:get_form).and_return(form)
         end
 
-        context "case logs that are owned or managed by your organisation" do
-          before do
-            sign_in user
+        context "a user that is not signed in" do
+          it "does not let you get case log tasklist pages you don't have access to" do
```

Comment:
```suggestion
        context "a user that is not signed in" do
          it "does not let the user get case log tasklist pages they don't have access to" do
```

otherwise it might read as if a user can block someone else from accessing tasklist pages